src/Components/Authorization/AuthForm.tsx

```diff
@@ -0,0 +1,109 @@
+import React from "react"
+
+import ForgotPasswordForm from "./ForgotPasswordForm"
+import LoginForm from "./LoginForm"
+import RegisterForm from "./RegisterForm"
+import { FormComponentType, FormFields, Mode } from "./Types"
+
+type Validator = (val: string) => string
+
+interface Props extends React.HTMLProps<HTMLDivElement> {
+  mode: Mode
```

Comment:
I'd prefer to use a more generalized router rather than implementing our own router. Have you looked into react-router's [`<HashRouter>`](https://reacttraining.com/react-router/web/api/HashRouter) or [`<MemoryRouter>`](https://reacttraining.com/react-router/web/api/MemoryRouter)?

Reply:
There are a few examples of React Router in Force, too, notably in the /Consignments app

Reply:
wow, it never occurred to me that this is something that would be _routed_ in that way.

Reply:
Good point, react router would make sense in terms of switching between various forms